tests/floating-point/fft/implementations/fft-pipelined.fil

```diff
@@ -43,20 +44,18 @@ comp PeaseFFT[NStages, N]<'G: L>(
   twiddle := new TwiddleFactor[NPoints, latency]<'G>();
 
   // contains all the twiddle stage data as an allways-alive constant matrix
-  bundle twiddle_in[NStages*NPoints]: for<_> ['G, 'G + latency] W;
+  bundle twiddle_in[NStages][NPoints/2][2]: ['G, 'G + latency] W;
 
   for twiddle_i in 0..NStages {
-    let start = twiddle_i * NPoints;
-    let end = (twiddle_i + 1) * NPoints;
-    twiddle_stage := new TwiddleStage[NStages, twiddle_i, W, latency]<'G>(twiddle.out{0..NPoints});
-    twiddle_in{start..end} = twiddle_stage.out{0..NPoints};
+    twiddle_stage := new TwiddleStage[NStages, twiddle_i, W, latency]<'G>(twiddle.out{0..NPoints/2}{0..2});
+    twiddle_in{twiddle_i}{0..NPoints/2}{0..2} = twiddle_stage.out{0..NPoints/2}{0..2};
   }
 
   // Synchronized data (happens between each stage, inputs and outputs to stride permutation)
   // Separated by the input and output to and from the registers.
   // There are Nstages of these (one for input and one for output for each stage)
-  bundle sync_data_in[(NStages + 1) * NPoints * 2]: for<sdii> ['G + sdii/(NPoints*2) * stage_latency, 'G + sdii/(NPoints*2) * stage_latency + 1] W;
-  bundle sync_data_out[NStages * NPoints * 2]:  for<sdoi> ['G + (sdoi/(NPoints*2)+1) * stage_latency, 'G + (sdoi/(NPoints*2)+1) * stage_latency + 1] W;
+  bundle sync_data_in[NStages+1][NPoints][2]: for<sdii, _sdi, __sdi> ['G + sdii*stage_latency, 'G + sdii*stage_latency + 1] W;
```

Comment:
You don't need to bind unused indices. You can just do `for<sdii>` and ignore the other two.